src/esn.calendar.libs/app/services/fullcalendar/planning-render-event.service.js

```diff
@@ -46,6 +47,30 @@ function calFullCalendarPlanningRenderEventService(
 
         tr.insertBefore(dot, tds[0]);
       }
+
+      function addStyleForEvent(calendar, element, event) {
```

Comment:
The method name and her content are not easy to understand, so a jsdoc to expalin the objective, params and the output is cool!